src/panels/lovelace/cards/hui-heading-card.ts

```diff
@@ -181,10 +181,13 @@ export class HuiHeadingCard extends LitElement implements LovelaceCard {
         min-width: 0;
       }
       .content.subtitle {
-        color: var(--secondary-text-color);
-        font-size: 14px;
-        font-weight: 500;
-        line-height: 20px;
+        color: var(
+          --ha-heading-card-subtitle-color,
+          var(--secondary-text-color)
+        );
+        font-size: var(--ha-heading-card-subtitle-font-size, 14px);
+        font-weight: var(--ha-heading-card-subtitle-font-weight, 500);
```

Comment:
Subtitle has higher weight than normal header? 🤔

Reply:
Yeah, it's less visible because the color is lighter and the size is smaller.